Added the review for the Che 7 End-user guide once again. (#783)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_importing-a-vs-code-extension-in-the-che-theia-editor.adoc

. Navigate to the link:https://marketplace.visualstudio.com/vscode[VS Code marketplace], locate the required extension, and copy the *VS Code Quick Open* code to the clipboard.

> . Open the *Che Plugins* panel in the Che-Theia IDE. If the panel is not shown, use the *View* -> *Che Plugins* menu options from the main menu.

Reviewer: ```suggestion
. Open the *Che Plugins* panel in the Che-Theia IDE. Alternatively, use the *View* -> *Che Plugins* option from the main menu.
```

Author: @rkratky , this is not an alternate step. If the panel isn't visible, the user can view it using the main menu options from the second sentence. WDYT?

+

> For macOS users, paste the install command, or the install button into the *Che Plugins* panel.

Reviewer: ```suggestion
For macOS users, paste the install command or the install button into the *Che Plugins* panel.
```
